pkg/api/defaults_test.go

```diff
@@ -1109,6 +1126,21 @@ func TestDistroDefaults(t *testing.T) {
 			AKSUbuntu1804,
 			AKSUbuntu1804,
 			true,
+			false,
+			AzurePublicCloud,
+		},
+		{
+			"default_kubernetes_usgov",
+			OrchestratorProfile{
+				OrchestratorType: Kubernetes,
+			},
+			AKS1604Deprecated,
```

Comment:
Why does this scenario get the deprecated aks distro?

Reply:
oh this is because this test is too cute and doesn't declare the keys

Reply:
yeah that's not the expected output, that's the input